config/hubs/utoronto.cluster.yaml

```diff
@@ -0,0 +1,108 @@
+name: utoronto
+provider: kubeconfig
+kubeconfig:
+  file: secrets/utoronto.yaml
+support:
+  config:
+    prometheus:
+      server:
+        resources:
+          requests:
+            cpu: 0.5
+            memory: 4Gi
+          limits:
+            cpu: 2
+            memory: 16Gi
+    grafana:
+      ingress:
+        hosts:
+          - grafana.utoronto.2i2c.cloud
+hubs:
+  - name: staging
+    domain: staging.utoronto.2i2c.cloud
+    template: basehub
+    auth0:
+      enabled: false
+    config: &utorontoHubConfig
+      jupyterhub:
+        custom:
+          homepage:
+            templateVars:
+              org:
+                name: University of Toronto
+                logo_url: https://raw.githubusercontent.com/utoronto-2i2c/homepage/master/extra-assets/images/home-hero.png
+                url: https://www.utoronto.ca/
+              designed_by:
+                name: 2i2c
+                url: https://2i2c.org
+              operated_by:
+                name: 2i2c
+                url: https://2i2c.org
+              funded_by:
+                name: University of Toronto
+                url: https://www.utoronto.ca/
+        singleuser:
+          image:
+            name: quay.io/2i2c/utoronto-image
+            tag: 83a724f5b829
+        scheduling:
+          userPlaceholder:
+            enabled: false
+            replicas: 0
+          userScheduler:
+            resources:
+              requests:
+                cpu: 0.1
+                memory: 512Mi
+              limits:
+                memory: 512Mi
+        proxy:
+          chp:
+            resources:
+              requests:
+                cpu: 0.1
+                memory: 128Mi
+              limits:
+                memory: 512Mi
+          traefik:
+            resources:
+              requests:
+                cpu: 0.1
+                memory: 256Mi
+              limits:
+                memory: 512Mi
```

Comment:
These are some resource limits from the old utoronto config. I wasn't sure whether to still use those or go with the default values in `basehub` template.

Reply:
IMO we should stick with the current UToronto configuration values

Reply:
Mmm... can I suggest otherwise, maybe? 😉 
I mean, do we know if these values are the proper ones for the current incantation of our hubs from this repo?
If we set the values without validation (and only because we inherit those values), we may be setting conditions that are not actually needed (nor optimal) and makes the whole thing more complex than it should be...
Maybe we can trace back why these values were needed and infer if we need to tweak them here as well before actually doing it?

Reply:
That was my feeling too @damianavila, esp since I believe the utoronto hub was the first version of our infra.

> Maybe we can trace back why these values were needed and infer if we need to tweak them here as well before actually doing it?

I will dig into it to try find out if these were tweaked afterwards or were part of the initial deployment.

Reply:
Ah - I was making an assumption here, let me re-phrase my suggestion:

- **If they have explicitly requested those limits** then we should keep them.
- **If they have not and this was just set by us without explicit rationale for special-casing it** then we should just go with whatever our other configurations use

Reply:
Looking at the commit history, it seems that the limits were set by us.
But I'm not 100% sure this means that there wasn't a request from utoronto or a limitation observed at some point. 

l would love if @yuvipanda could confirm this :eyes: 

This is a side-by-side resource differences. Hope this makes it more easier to follow
![resource-diffs](https://user-images.githubusercontent.com/7579677/144663454-26a24a5f-4b1c-4e8d-9788-5bca44c2c6ab.png)